panel/chat/langchain.py

```diff
@@ -162,3 +162,16 @@ def on_retriever_end(self, documents, **kwargs: Any) -> Any:
     def on_text(self, text: str, **kwargs: Any):
         """Run when text is received."""
         return super().on_text(text, **kwargs)
+
+    def on_chat_model_start(
+        self,
+        serialized: Dict[str, Any],
+        messages: List,
+        **kwargs: Any
+    ) -> None:
+        """
+        Does not do anything here; the other methods will handle output,
+        primarily on_llm_new_token and on_llm_end. Note, this method
+        explicitly does nothing and does not call super() to prevent crashing into
+        a NotImplementedError from the inherited class.
```

Comment:
```suggestion
        Does not do anything to prevent the inherited class from raising
        NotImplementedError, and thus will not call super() here;
        the other methods will handle output, primarily
        `on_llm_new_token` and `on_llm_end`.
```